src/components/FilterPanel/FilterAccordion/FilterAccordion.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file Filter accordion component.
+ * @copyright IBM Security 2020
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { Accordion } from '../../Accordion';
+import { getComponentNamespace } from '../../../globals/namespace';
+import FilterGroup from '../FilterGroup';
+
+const namespace = getComponentNamespace('filter-accordion');
+
+const FilterAccordion = ({
```

Comment:
What do y'all think of these subcomponent names? I think `FilterPanelAccordion` would make more sense, but it's just so long. Especially for the `FilterAccordionItem` which would be `FilterPanelAccordionItem`. However, I see that the `SummaryCard` subcomponents all have `SummaryCard` in front of them. So maybe length doesn't matter?

Reply:
Honestly I don't particularly mind the additional length, especially if it adds clarity 👍

Reply:
Okay, I'll replace all `Filter...` with `FilterPanel...`